test/unit/test_formula.py

```diff
@@ -63,6 +76,42 @@ def test_generate_formula_no_article_description():
     }
 
     formula = Formula.generate_formula_data(USERNAME, REPO_NAME, repository, CHECKSUM, INSTALL, TAR_URL)
-    cassette_filename = 'test_formula_template_no_article_description.rb'
+    cassette_filename = 'test_formula_template_no_depends_on.rb'
```

Comment:
The `cassette_filename` should match the test name, in this case it should be `test_formula_template_no_article_description`

Reply:
Of course, sorry for this mistake :)